src/styles/views/UserHand.scss

```diff
@@ -0,0 +1,40 @@
+@import 'src/styles/theme';
+
+.hand {
+  &.main-container{
+    margin-top: 2em;
+    margin-bottom: 20px;
+    min-height: 300px;
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-evenly;
+    align-items: flex-start;
+    align-content: center;
+    gap: 20px;
+  }
+  &.card-container {
+    &.white {
+      min-width: 250px;
+      max-width: 1050px;
+    }
+    &.black {
+      width: 250px;
+    }
+    justify-content: center;
+    align-items: center;
+    display: flex;
+    flex-flow: column wrap;
+  }
+
+  &.text{
+    color: $background;
```

Comment:
confusing, don't you want to use ``$WHITE_CARD_TEXT_COLOR`` ?

Reply:
It is used for "your White Cards" and "Your Black Cards" and not part of the card